pkg/cvo/updatepayload.go

```diff
@@ -315,3 +317,15 @@ func findUpdateFromConfigVersion(config *configv1.ClusterVersion, version string
 	}
 	return configv1.Update{}, false
 }
+
+func (optr *Operator) clusterProfileRetriever() (string, error) {
+	configMap, err := optr.cmConfigLister.Get(internal.ClusterProfileConfigMap)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			klog.Infof("Cannot get ConfigMap %s/%s, using default profile: %v", internal.ConfigNamespace, internal.ClusterProfileConfigMap, err)
+			return "default", nil
```

Comment:
Per [this][1], I think we shoulf `return "", nil` here.

[1]: https://github.com/openshift/cluster-version-operator/pull/404#discussion_r461555262

Reply:
done! Now there is no more mention of the default profile, it's simpler.